.travis.yml

```
@@ -3,6 +3,11 @@ node_js:
   - "10"
   - "8"
   - "6"
+install:
+  - npm install -g codecov
```

```suggestion
  - travis_wait npm ci
  - npm install -g codecov
```

Default install is `npm install` and if lock files exist then `npm ci`.  Build is failing because we added `install` script , which does not have `npm install` or `npm ci`.

---

@hemal7735 Thanks for pointing this out! 👍 Added the suggestion. Build failed with `npm ci` on v6. Trying `npm run ci` instead (failed too).

Do you have an idea?

---

`npm ci` should work. Failures on v6 might be temporary. `npm run ci` should not work, as `npm run <command>` will be picked up from package.json.

---

@hemal7735 It seems `npm ci` was only introduced in v5.7.0 https://blog.npmjs.org/post/171139955345/v570

What do you think about adding?
```
before_install:
  - npm i -g npm
```

---

```
before_install:
    - npm install -g npm@latest
```
?